GramAddict/core/utils.py

```diff
@@ -243,5 +243,39 @@ def validate_url(x):
         return False
 
 
+def read_file(filename):
+    if not filename.lower().endswith(".txt"):
+        filename = f"{filename}.txt"
+    if os.path.isfile(filename):
+        with open(filename, "r") as f:
+            return f.readlines()
```

Comment:
Similar to the other comment, this is bad practice too. If it's a huge file (unlikely) this would fill up ram. This should be read and processed in a different manner.